pkg/database/postgresql.go

```diff
@@ -11,17 +11,33 @@ import (
 	"slices"
 
 	"github.com/huandu/go-sqlbuilder"
-	"github.com/jmoiron/sqlx"
 	"github.com/kubearchive/kubearchive/pkg/database/facade"
 	"github.com/lib/pq"
 	_ "github.com/lib/pq"
 )
 
 func init() {
-	RegisteredDatabases["postgresql"] = NewPostgreSQLDatabase
+
+	RegisteredDatabases["postgresql"] = NewPostgreSQLDatabase()
 	RegisteredDBCreators["postgresql"] = NewPostgreSQLCreator
 }
 
+type PostgreSQLDatabase struct {
```

Comment:
same as previous comment

```suggestion
type postgreSQLDatabase struct {
```

Reply:
Applied!